arb_os/evmCallStack.mini

```diff
@@ -792,9 +847,13 @@ public impure func evmCallStack_returnFromCall(
                 topFrame.storageDelta - int(storageReclaimedForSDQ(topFrame)),  // won't underflow because signed subtraction
                 globalCurrentTxRequest.feeStats
             )?;
+
             if (completed) {
                 if (globalCurrentTxRequest.isConstructor) {
-                    // Constructor call was successful, so finish setting up the account
+                    // Constructor call was successful, so finish setting up the account.
+                    // The constructor's returndata is EVM code for the contract.
+                    // Compile that EVM code into AVM, and substitute that in as the code for the contract.
+                    let (entryPoint, evmJumpTable, _) = translateEvmCodeSegment(bytestream_new(returnData), false);
```

Comment:
I think this is a regression introduced while merging

Reply:
Fixed.  Good catch!

Reply:
Fixed.  Good catch.